staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -55,7 +55,9 @@ const (
 	flagHTTPCacheDir     = "cache-dir"
```

Comment:
should this be `flagCacheDir` now? And also update below: https://github.com/kubernetes/kubernetes/pull/92910/files#diff-4e28bfb0f0c95fb4bd5a77134c7ad3f1L256